### DISCUSSION TITLE
Deprecate `beat` display option in Time & Date

### DISCUSSION THREAD

**Discussion by andrewjswan** [home-assistant/architecture#1037] [General] [answered]
**Deprecate `beat` display option in Time & Date** - what’s wrong with this sensor? 
If it’s not needed, people don’t add it, and if people use it, they add it. Do a couple of lines of code really interfere with the HA architecture that much?

Of course, it can be replaced with a template sensor, but all the sensors of this integration can also be replaced with a template sensor, then why do we leave the others and remove this one?

If a person does not need this sensor, he does not add it to his configuration and does not use it. But if you need it, then it is more convenient to add it through the GUI than to create a template sensor that will not provide the required accuracy.

**Comment by frenck** [comment]
I'm not sure if this is an architectural proposal/discussion?

**Reply by andrewjswan** [reply]
Can you tell me where to discuss this? The forum is useless. Open Issue? Or?

**Reply by frenck** [reply, marked as answer]
The forums is the place for support, feature requests and other general discussions. In case you have a bug, our issue tracker is the place to go.

The specific place you are now in, is for developers discussing fundamental architectural changes to the core architecture (like our entity models).

../Frenck

**Comment by andrewjswan** [comment]
https://github.com/home-assistant/core/issues/109963
